index.html

```diff
@@ -1,14 +1,164 @@
+            <br>
+          </div>
+          <br>
+          <div class="date-container2">
+            <div class="date">JULY 24</div>
+            <h5 class="sidebar-post-title">ONE MORE POST</h5>
+            <p class="sidebar-post">Lorem ipsum dolor sit amet consectetur adipisicing elit. Officiis atque magnam minima, corporis quibusdam et doloribus ipsum ad cum expedita!</p>
+            <br>
+          </div>
+          <div class="date-container">
+            <h5 class="sidebar-post-title">SOMETHING OF NOTE</h5>
+            <div class="sidebar-body">
+             <img class="sidebar-pics" src="https://picsum.photos/300/204"/><p class="sidebar-post">Lorem ipsum dolor sit amet consectetur adipisicing elit. Explicabo vel ad quod atque quos natus saepe exercitationem consequatur odit dolores? Mollitia consequatur reprehenderit earum necessitatibus, magnam beatae. Quae ratione odio suscipit harum, iusto repellendus inventore!</p>
+              <br>
+            </div>
+            <button class="button3" type="button"><a target="_blank" href="learn-more.html">LEARN MORE</a></button>
+            <br>
+          </div>
+
+          <h5 class="sidebar-post-title">SOMETHING OF LESS NOTE</h5>
+          <div class="sidebar-body">
+            <img class="sidebar-pics" src="https://picsum.photos/300/205"/><p class="sidebar-post">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Cumque voluptate incidunt non, ullam dicta, nihil sequi, recusandae impedit voluptates aliquid ad! Modi placeat dignissimos molestiae obcaecati, odit maiores voluptatem ex, quam nobis perspiciatis explicabo eveniet.</p>
+          </div>
+          <br>
+          <button class="button3" type="button"><a target="_blank" href="learn-more.html">LEARN MORE</a></button>
+        </div>
+      </div>
+    
+<!--Footer-->
+
+    <section id="footer">  
+
+      <div id="footer-container-top">
+        <div class="footer-title">
+          <h3 class="questions">QUESTIONS OR COMMENTS?</h3> <h3 class="get">GET IN TOUCH:</h3>
+        </div>
+      </div>
+
+      <div id="footer-container-bottom">
+        <div class="footer-left-column">
+          <div class="footer-content">
+            <div class="name-email">
+              <input id="name" name="name" rows="1" cols="5" type="text" placeholder="Name"/>
+            &nbsp;
+              <input id="email" name="email" rows="1" cols="5" type="text" placeholder="Email"/>
+            </div>
+          <br>
+          <br>
+            <div class="message">
+              <textarea id="message" name="message" rows="10" cols="20" placeholder="Message"></textarea>
+            <br>
+              <button class="button4" type="button"><img class="mail-img" src="file:///Users/jaysondurante/Desktop/Jayson/Burlington%20Code%20Academy/Remock%20Proj%20Pics/footerlink-email.png"/>SEND MESSAGE</button>

```

Comment:
Try to use more descriptive class names while naming your classes. This could have been something like submitBtn or something similar. We also are having the same issues as at the top of this file where your image source is a local file on your computer